src/sentry/profiles/profile_chunks.py

```diff
@@ -35,7 +35,8 @@ def get_chunk_ids(
             Condition(Column("project_id"), Op.EQ, project_id),
             Condition(Column("profiler_id"), Op.EQ, profiler_id),
         ],
-        orderby=[OrderBy(Column("start_timestamp"), Direction.ASC)],
+        # We want the generate the flamegraph using the newest data
```

Comment:
@Zylphrex, small clarification: `get_chunk_ids` is only used for `flamechart` (merging several chunks into a unique one), not for flamegraph. The idea of using DESC was to start from the oldest data so that we could then move forward as we added pagination support (assuming continuous scrolling is something we're still interested in supporting).

Anyway, since we don't have pagination yet, I think getting the most recent data is the right call. Just wanted to clarify.

Reply:
Yep, going to leave this for now to prefer newer chunks for now. We'll think about pagination when we get there.